curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/create-a-bulleted-unordered-list.english.md

```diff
@@ -32,7 +32,8 @@ tests:
     testString: assert(code.match(/<\/ul>/gi) && code.match(/<ul/gi) && code.match(/<\/ul>/gi).length === code.match(/<ul/gi).length, 'Make sure your <code>ul</code> element has a closing tag.');
   - text: Make sure your <code>li</code> elements have closing tags.
     testString: assert(code.match(/<\/li>/gi) && code.match(/<li[\s>]/gi) && code.match(/<\/li>/gi).length === code.match(/<li[\s>]/gi).length, 'Make sure your <code>li</code> elements have closing tags.');
-
+  - text: Make sure your <code>li</code> elements doesn’t contain empty string or only whitespace.
+    testString: assert($("ul li").filter(function(index){return $(this).text().trim().length === 0;}).length === 0, 'Make sure your <code>li</code> elements doesn\’t contain empty string or only whitespace.');
```

Comment:
While your added test works fine, yo could use the same logic but simplify a bit.
```suggestion
    testString: assert($("ul li").filter((_, item) => !$(item).text().trim()).length === 0, 'Make sure your <code>li</code> elements don\’t contain an empty string or only white-space.');
```